tools/cldr-apps/js/src/esm/cldrAccount.mjs

```diff
@@ -354,6 +356,10 @@ function getTableEnd(json) {
   return html;
 }
 
+function canListMultipleUsers() {
+  return !!cldrStatus.getPermissions()?.userIsManager;
```

Comment:
another way would be to expand the `cldrStatus.getPermissions()` struct ( a string -> boolean map i think) to include a new item from the server side.

Reply:
Good idea. I'll follow up with a PR to get canDoList from the back end...